frontend/src/components/molecules/MatchItem/MatchItem.tsx

```diff
@@ -14,15 +14,16 @@ import MatchWinnerDeterminationDialog from "../MatchWinnerDeterminationDialog/Ma
 import {MatchStatus} from "../../atoms/MatchStatus";
 
 export type MatchItemProps = {
-   onClickTeam: () => void,
+   onClickTeam: (matchId: string, teamName: string) => void,
```

Comment:
Mówiliśmy, że nie ma teamName, to co się pod tym kryje?

Reply:
Ale jaja, dzięki za uwagę!!! Dobrze, że to zauważyłaś, ja już odruchowo to piszę jako ID i zapomniałem zmienić!